src/main/java/pro/quicksense/modules/util/SecurityConfig.java

```diff
@@ -0,0 +1,22 @@
+package pro.quicksense.modules.util;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .csrf(csrf -> csrf.disable())
+                .authorizeHttpRequests(authz -> authz
+                        .requestMatchers("/user/register", "/user/login").permitAll()
+                        .anyRequest().authenticated()
+                );
+        return http.build();
+    }
+}
```

Comment:
I think we don't need to set CSRF here. We will have a proxy in front of the frontend and backend services to provide the same entrypoint.